core/src/main/java/org/infinispan/commands/topology/CacheLeaveCommand.java

```diff
@@ -20,43 +20,38 @@ public class CacheLeaveCommand extends AbstractCacheControlCommand {
    public static final byte COMMAND_ID = 86;
 
    private String cacheName;
-   private int viewId;
 
    // For CommandIdUniquenessTest only
    public CacheLeaveCommand() {
       super(COMMAND_ID);
    }
 
-   public CacheLeaveCommand(String cacheName, Address origin, int viewId) {
+   public CacheLeaveCommand(String cacheName, Address origin) {
       super(COMMAND_ID, origin);
       this.cacheName = cacheName;
-      this.viewId = viewId;
    }
 
    @Override
    public CompletionStage<?> invokeAsync(GlobalComponentRegistry gcr) throws Throwable {
       return gcr.getClusterTopologyManager()
-            .handleLeave(cacheName, origin, viewId);
+            .handleLeave(cacheName, origin);
    }
 
    @Override
    public void writeTo(ObjectOutput output) throws IOException {
       MarshallUtil.marshallString(cacheName, output);
-      output.writeInt(viewId);
```

Comment:
should we keep `output.writeInt(0)/input.readInt()` as an attempt to keep backward compatibility between minor releases?